ui/pages/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -122,6 +122,8 @@ const mapStateToProps = (state, ownProps) => {
   const toEns = ensResolutionsByAddress[checksummedAddress] || '';
   const toNickname = addressBookObject ? addressBookObject.name : '';
   const transactionStatus = transaction ? transaction.status : '';
+  const supportsEIP1599 =
```

Comment:
It looks like this was originally a typo, should this be `supportsEIP1559` (not `supportsEIP1599`)?